docker/build_testing_sheet.dockerfile

```diff
@@ -0,0 +1,19 @@
+FROM ubuntu:bionic
+
+RUN apt-get update -y
+
+COPY . /kolibri
+
+COPY ./kolibri/.buildkite $GOOGLE_SPREADSHEET_CREDENTIALS
+
+CMD cd /kolibri && \
+    pip install -r requirements/testing_sheet.txt
+
+VOLUME /kolibridist/
+
+CMD export GOOGLE_SPREADSHEET_CREDENTIALS=./kolibri/.buildkite/*.json && \
```

Comment:
Docker's `run` command lets you do this, and includes checks for variable definition
https://docs.docker.com/engine/reference/commandline/run/#set-environment-variables--e---env---env-file

Reply:
I'll try this thanks.